### PR TITLE
Database works now

### PR DESCRIPTION
**Pull request by runtilov**
The server can only get the list of admins  from the database and an admin to the db. We need the definition of the API to completely integrate the database.

### CODE REVIEW

File: src/migrations/1-create-admin.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+module.exports = {
```
Comment (alxolr) [inline src/migrations/1-create-admin.js]: Please create migrations using sequelize cli not by hand

File: src/models/course.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    name: {
+      type: DataTypes.STRING,
+    },
+
+  }, {
+    tableName: 'courses',
+    timestamps: false,
+  });
+  Course.associate = (models) => {
+    // Course.hasMany(Groups, {foreignKey: 'idSpecialities'});
```
Comment (alxolr) [inline src/models/course.js]: Commented code should not be commited

File: src/seeders/1-demo-admin.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
```
Comment (alxolr) [inline src/seeders/1-demo-admin.js]: generate seeds by using sequelize cli

File: src/seeders/1-demo-admin.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert('admins', [{
+      first_name: 'FirstSeed',
+    }], {});
+  },
+
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.bilkDelete('admins', null, {});
```
Comment (alxolr) [inline src/seeders/1-demo-admin.js]: Not sure `bilkDelete` exists

File: src/seeders/2-demo-specialities.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert('specialities', [{
+      name: 'FirstSeedSpec',
+    }], {});
+  },
+
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.bilkDelete('specialities', null, {});
```
Comment (alxolr) [inline src/seeders/2-demo-specialities.js]: Copy paste detected

File: config/config.json [hunk 6]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (alxolr) [inline config/config.json]: You can add in `sequelizerc` to use the default.js config, and then in index.js in models to change the connection to be the proper one

File: src/seeders/1-demo-admin.js (hunk 3 above)
Comment (runtilov) [reply src/seeders/1-demo-admin.js]: we used sequelize cli, we just renamed them

**Pull request merged**